userspace/libsinsp/CMakeLists.txt

```diff
@@ -310,3 +310,27 @@ if (BUILD_LIBSINSP_EXAMPLES)
 	add_subdirectory(examples)
 endif()
 
+# Map each library name to its .pc file.
+set(cares_pc_module libcares)
+set(curl_pc_module libcurl)
+set(grpc_pc_module grpc)
+set(jsoncpp_pc_module jsoncpp)
+set(luajit_pc_module luajit)
+set(openssl_pc_module openssl)
+set(protobuf_pc_module protobuf)
+set(tbb_pc_module tbb)
```

Comment:
We just added a dependency to `re2`, I think it should be added here.

Reply:
Done. Thanks!